### ISSUE TITLE
Is this plugin abandoned?

### ISSUE THREAD

**Issue by hughbris** [Stepanov-Sergey/GFFI#3]
I noticed this repo searching for "gallery" in the Grav plugin repo, five years since last update and description of "test".

I'm loosely following the [Abandoned Resource Protocol](https://learn.getgrav.org/17/advanced/grav-development#abandoned-resource-protocol) with a view to removing this presumably unmaintained plugin.

Has development of this plugin been abandoned?

Thanks :)

**Comment by Stepanov-Sergey** [comment]
you are right, work with the plugin has been discontinued and is not planned in the future

**Comment by diomed** [comment]
you could archive repository
